Adding new faculty member Anupam Das at NC State University

anupam-das opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

anupam-das committed:
adding myself a new faculty member of NC State univ. and fixing a wrong google scholar userid for someone with the same name

anupam-das committed:
Revert "adding myself a new faculty member of NC State univ. and fixing a wrong google scholar userid for someone with the same name"

This reverts commit dc60224b98fe72f378126fc583f57f1f134464e7.

reverting the original csranking.csv file as certain unicode characters with messed up

anupam-das committed:
adding myself a new faculty member of NC State univ. and fixing a wrong google scholar userid for someone with the same name

anupam-das commented:
Just to make sure the information I provided is correct, please visit the following link to our departmental lab webpage.

https://wspr.csc.ncsu.edu/people.html

emeryberger commented:
Please delete the second (Lyon) entry and contact the DBLP folks to create a proper page for you. Thanks.

anupam-das commented:
The second entry is someone else with the same (exact) name, so don't know if I should delete it. I had already requested DBPL to create a proper page for me and the proper page for me is available at https://dblp.org/pers/hd/d/Das_0001:Anupam

anupam-das committed:
adding disambiguation suffix (0001) to link to the proper DBPL page (https://dblp.org/pers/hd/d/Das_0001:Anupam)

anupam-das commented:
Updated both entries after DBLP created proper pages for both faculty members. I have updated the `csrankings.csv` file with the proper DBLP suffixes.

anupam-das committed:
adding disambiguation suffix (0002) to link to the proper DBPL page for a research with the same exact name (after DBLP create proper pages)
